src/organizations/components/OrgProfileTab.tsx

```diff
@@ -43,105 +49,175 @@ const OrgProfileTab: FC = () => {
-                text={me.id}
-                testID="copy-btn--userid"
-                onCopy={generateCopyText('User ID', me.id)}
-              />
-              <label className="code-snippet--label">
-                {`${me.name} |`} <b>User ID</b>
-              </label>
-            </FlexBox>
-          </div>
+          <FlexBox direction={FlexDirection.Row} margin={ComponentSize.Medium}>
+            <Input
+              value={org.name}
+              style={{width: 'max-content'}}
+              data-testid="danger-zone--org-name"
+              testID="danger-zone--org-name"
+            ></Input>
+            <Button
+              testID="rename-org--button"
+              text="Rename"
+              icon={IconFont.Pencil}
+              onClick={handleShowEditOverlay}
+            />
+          </FlexBox>
+          <FlexBox
+            direction={FlexDirection.Row}
+            margin={ComponentSize.Medium}
+            justifyContent={JustifyContent.SpaceBetween}
+            stretchToFitWidth={true}
+            style={{width: '85%'}}
+          >
+            {[
+              {
+                label: 'Provider',
+                src: convertProviderSymbol(me.quartzMe?.billingProvider),
+              },
+              {label: 'Region', src: me.quartzMe?.regionCode},
+              {label: 'Location', src: me.quartzMe?.regionName},
+            ].map(({label, src}) => {
+              if (!!src) {
+                return (
+                  <FlexBox
+                    key={`org-${label.toLowerCase()}`}
+                    direction={FlexDirection.Column}
+                    margin={ComponentSize.Large}
+                    alignItems={AlignItems.FlexStart}
+                  >
+                    <Heading
+                      element={HeadingElement.H4}
+                      weight={FontWeight.Regular}
+                      style={headingStyle}
+                    >
+                      {label}
+                    </Heading>
+                    <span style={{fontWeight: FontWeight.Light, width: '100%'}}>
+                      {src}
+                    </span>
+                  </FlexBox>
+                )
+              } else {
+                return null
+              }
+            })}
+          </FlexBox>
           <Heading
             element={HeadingElement.H4}
-            style={{marginTop: '16px', marginBottom: '4px'}}
+            style={headingStyle}
             weight={FontWeight.Regular}
           >
-            Organization ID
+            Cluster URL
           </Heading>
-          <div className="code-snippet" data-testid="code-snippet--orgid">
+          <div
+            className="code-snippet"
+            style={stretchStyle}
+            data-testid="code-snippet--userid"
+          >
             <div className="code-snippet--text">
               <pre>
-                <code>{org.id}</code>
+                <code>{me.quartzMe.clusterHost}</code>
```

Comment:
```suggestion
                <code>{me.quartzMe?.clusterHost}</code>
```

Also, what happens if this isn't provided by the API? How's it look?

Reply:
good call. this entire node (view element with code snippet) is now wrapped in an existence check.